doc/source/cluster/kubernetes/user-guides/helm-chart-rbac.md

```diff
@@ -0,0 +1,187 @@
+(kuberay-helm-chart-rbac)=
+
+# Helm Chart RBAC
+
+KubeRay utilizes [Kubernetes Role-Based Access Control (RBAC) resources](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to grant permissions for monitoring and managing resources.
+This document describes how to configure the KubeRay Helm chart to create RBAC resources for 3 different use cases.
+
+* [Case 1: Watch all namespaces in the Kubernetes cluster](case1-watch-all-namespaces)
+* [Case 2: Watch the namespace where the operator is deployed](case2-watch-1-namespace)
+* [Case 3: Watch multiple namespaces in the Kubernetes cluster](case3-watch-multiple-namespaces)
+
+## Parameters
+
+You can configure the KubeRay Helm chart to create RBAC resources for different use cases by modifying the following parameters in the [values.yaml](https://github.com/ray-project/kuberay/blob/master/helm-chart/kuberay-operator/values.yaml).
+Then, you can install the KubeRay Helm chart with the modified **values.yaml**.
+
+```shell
+# Step 1: Clone the KubeRay repository
+# Step 2: Modify the helm-chart/kuberay-operator/values.yaml
+# Step 3: Install the KubeRay Helm chart (path: helm-chart/kuberay-operator)
+helm install kuberay-operator .
+```
+
+* **`rbacEnable`**
+  * It determines whether to create RBAC resources. If set to false, the Helm chart doesn't create any RBAC resources.
+  * By default, it's set to `true` and creates RBAC resources.
+
+* **`singleNamespaceInstall`**
+  * It determines whether to create namespace-scoped or cluster-scoped RBAC resources. If set to true, the Helm chart creates Role and RoleBinding. Otherwise, it creates ClusterRole and ClusterRoleBinding instead.
+  * By default, it's set to `false` and creates cluster-scoped RBAC resources.
+
+* **`watchNamespace`**
+  * The KubeRay operator's informer watches the custom resources in the namespaces listed in the `watchNamespace` parameter.
+
+* **`crNamespacedRbacEnable`**
+  * In most cases, it should be `true`, unless you are using a Kubernetes cluster managed by GitOps tools such as ArgoCD. For additional details, refer to [ray-project/kuberay#1162](https://github.com/ray-project/kuberay/pull/1162).
+  * By default, it's set to `true`.
+
+The [values.yaml](https://github.com/ray-project/kuberay/blob/master/helm-chart/kuberay-operator/values.yaml) file contains detailed descriptions of the parameters.
+In addition, these pull requests, [ray-project/kuberay#1106](https://github.com/ray-project/kuberay/pull/1106), [ray-project/kuberay#1162](https://github.com/ray-project/kuberay/pull/1162), and [ray-project/kuberay#1190](https://github.com/ray-project/kuberay/pull/1190), may also be helpful to understand these parameters.
+
+(case1-watch-all-namespaces)=
+## Case 1: Watch all namespaces in the Kubernetes cluster
+
+![Watch all namespaces in the Kubernetes cluster](../images/rbac-clusterrole.svg)
+
+This is the KubeRay's default setting.
```

Comment:
```suggestion
This is KubeRay's default setting.
```

Reply:
We can fix this in a followup if there's a time constraint for this PR, since CI takes a while to run.

Reply:
> there's a time constraint for this PR

There is no time constraint for this PR. I will commit your suggestion to this branch.